tests/puppeteer/pages/FO/FO_cart.js

```diff
@@ -0,0 +1,35 @@
+const CommonPage = require('../commonPage');
+
+module.exports = class FO_CART extends CommonPage {
```

Comment:
```suggestion
module.exports = class Cart extends CommonPage {
```

Reply:
These modifications will be done in a separate PR.